server.js

```diff
@@ -25,8 +54,30 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get("/", (request, response) => {
```

Comment:
did you install // npm install express-list-endpoints and added
const listEndpoints = require('express-list-endpoints')

this would list your endpoint automatically after adding:
`app.get("/", (req, res) => {
  res.json(listEndpoints(app));
});`